### DISCUSSION TITLE
Conversion to onnx not working for the en_PP-OCRv4_rec_infer

### DISCUSSION THREAD

**Discussion by MadMenHitBooker** [PaddlePaddle/PaddleOCR#12667] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Linux
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：2.7
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no): no

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

Hi , i'm having issues trying to convert the model to onnx

```json
{
"url":
"https://paddleocr.bj.bcebos.com/PP-OCRv4/english/en_PP-OCRv4_rec_infer.tar",
"dict_path": "./ppocr/utils/en_dict.txt"
}
```

The dict associated is not the right one, so when i'm using the converter __paddleocr_convert__, it throwing some errors.
Can you check that this is the one , or do i need to transform the dict

**Comment by SWHL** [comment]
You can try the online convert demo → [link](https://huggingface.co/spaces/SWHL/PaddleOCRModelConverter)
